build/jest/jest.config.common.js

```diff
@@ -8,9 +11,12 @@ module.exports = {
     '^.+\\.(ts|tsx)$': 'ts-jest',
   },
   moduleNameMapper: {
-    'docs/(.*)$': `${__dirname}/../../../docs/$1`,
-    'src/(.*)$': `${__dirname}/../src/$1`,
-    'test/(.*)$': `${__dirname}/../test/$1`,
+    ...lernaAliases(),
+
+    // Legacy aliases, they should not be used in new tests
```

Comment:
just to be clear - it means that we've left it as a follow-up work (I mean, tests grooming), right? In that case, could you, please, create a note in the PR description (or do whatever is necessary), so we won't forget it